langstream-core/src/main/java/ai/langstream/impl/nar/NarFileHandler.java

```diff
@@ -47,6 +47,13 @@ public class NarFileHandler
                 AssetManagerRegistry.AssetManagerPackageLoader,
                 TopicConnectionsRuntimeRegistry.TopicConnectionsPackageLoader {
 
+    private static final boolean CLOSE_CLASSLOADERS =
+            Boolean.parseBoolean(System.getProperty("langstream.nar.closeClassloaders", "true"));
+
+    static {
+        log.info("langstream.nar.close-classloaders = {}", CLOSE_CLASSLOADERS);
```

Comment:
please align to the system property name

Reply:
good catch